config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -39,7 +39,7 @@
 # will have a different format that is not supported by Rails 6.1 applications.
 # Only change this value after your application is fully deployed to Rails 7.0
 # and you have no plans to rollback.
-# Rails.application.config.active_support.cache_format_version = 7.0
+Rails.application.config.active_support.cache_format_version = 7.0
```

Comment:
For more information see https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#new-activesupport-cache-serialization-format

Reply:
> Rails 7.0 is able to read both formats so the cache won't be invalidated during the upgrade.

Great to see this last bit in the docs! 😄

Reply:
Yes i should've included that tidbit to save reviewers sometime :sweat_smile: